tests/unifiedPlatform/platform/platform_test.go

```diff
@@ -55,3 +58,37 @@ var _ = Describe("{PlatformOnboardingTest}", func() {
 		defer EndTorpedoTest()
 	})
 })
+
+var _ = Describe("{TestRbacForPds}", func() {
```

Comment:
If this is a dummy test please move this to tests/unifiedPlatform/platform/platform_test_dummy.go